smbutil.c

```diff
@@ -866,16 +868,18 @@ smb_fdata(netdissect_options *ndo,
 		return(buf);
 	    memset(s, 0, sizeof(s));
 	    p = strchr(fmt, ']');
+	    if (p == NULL)
+			return(NULL);
 	    if ((size_t)(p - fmt + 1) > sizeof(s)) {
-		/* overrun */
-		return(buf);
+			/* overrun */
```

Comment:
These two changes don't need to be done.  The indentation style is 4-space indentation, with tabs used for 8-space-character sequences.